redirects.js

```diff
@@ -52,6 +52,44 @@ const setup = (app) => {
   conflates a language part ("zh") with a region ("europe") so we want to keep zh out of the manifest JSON */
   app.route("/ncov/zh").get((req, res) => res.redirect("/ncov/global/zh"));
 
+  // DOCS, HELP Redirects for docs migration:
+  const mainReadTheDocs = "https://docs.nextstrain.org/en/latest";
+  const augurReadTheDocs = "https://docs.nextstrain.org/projects/augur/en/migrate-docs";
+
+  const docsRedirects = {
+    "/docs": `${mainReadTheDocs}`,
+    "/help": `${mainReadTheDocs}/learn/about-nextstrain.html`,
+    "/docs/getting-started/introduction": `${mainReadTheDocs}/learn/about-nextstrain.html`,
+    "/docs/getting-started/local-installation": `${mainReadTheDocs}/guides/install/local-installation.html`,
+    "/docs/getting-started/local-vs-container-install": `${mainReadTheDocs}/guides/install/index.html`,
+    "/docs/getting-started/quickstart": `${mainReadTheDocs}/tutorials/quickstart.html`,
+    "/docs/getting-started/windows-help": `${mainReadTheDocs}/guides/install/windows-help.html`,
+    "/docs/bioinformatics/introduction-to-augur": `${augurReadTheDocs}/faq/introduction-to-augur.html`,
+    "/docs/bioinformatics/what-is-a-build": `${augurReadTheDocs}/faq/what-is-a-build.html`,
+    "/docs/bioinformatics/data-formats": `${mainReadTheDocs}/reference/formats/data-formats.html`,
+    "/docs/tutorials/defining-clades": `${mainReadTheDocs}/guides/bioinformatics/defining-clades.html`,
+    "/docs/tutorials/zika": `${mainReadTheDocs}/tutorials/zika.html`,
+    "/docs/contributing/community-builds": `${mainReadTheDocs}/guides/share/community-builds.html`,
+    "/docs/contributing/documentation": `${mainReadTheDocs}/guides/contribute/documentation.html`,
+    "/docs/contributing/fetch-data-from-custom-urls": `${mainReadTheDocs}/guides/share/fetch-via-urls.html`,
+    "/docs/contributing/nextstrain-groups": `${mainReadTheDocs}/guides/share/nextstrain-groups.html`,
+    "/docs/contributing/sharing-data": `${mainReadTheDocs}/guides/share/index.html`,
+    "/docs/visualisation/download-data": `${mainReadTheDocs}/guides/share/download-data.html`,
+    "/docs/visualisation/map-interpretation": `${mainReadTheDocs}/learn/interpret/map-interpretation.html`,
+    "/help/general/about-nextstrain": `${mainReadTheDocs}/learn/about-nextstrain.html`,
+    "/help/general/how-to-read-a-tree": `${mainReadTheDocs}/learn/interpret/how-to-read-a-tree.html`,
+    "/help/general/interacting-with-nextstrain": `${mainReadTheDocs}/learn/interpret/interacting-with-nextstrain.html`,
+    "/help/coronavirus/FAQ": `${mainReadTheDocs}/learn/pathogens/coronavirus/FAQ.html`,
+    "/help/coronavirus/SARS-CoV-2": `${mainReadTheDocs}/learn/pathogens/coronavirus/SARS-CoV-2.html`,
+    "/help/coronavirus/Technical-FAQ": `${mainReadTheDocs}/learn/pathogens/coronavirus/Technical-FAQ.html`,
+    "/help/coronavirus/human-CoV": `${mainReadTheDocs}/learn/pathogens/coronavirus/human-CoV.html`
+  };
```

Comment:
The redirects from https://github.com/nextstrain/nextstrain.org/blob/master/static-site/redirects.json should be shifted to here, which is consistent with our plan to remove all "docs" and "help" content from the Gatsby section of the site.

Reply:
I am adding entries for all those redirects trying to match values from the gatsby mapping with keys in the new mapping I have here. e.g. for the entry in the gatsby redirects:
>   "/about/overview/introduction": "/docs/getting-started/introduction",

I look for any existing main redirect with the key "/docs/getting-started/introduction":
> "/docs/getting-started/introduction": `${mainReadTheDocs}/learn/about-nextstrain.html`,

and add the following entry to the main redirects list (with the goal of reducing double redirects):
> "/about/overview/introduction": `${mainReadTheDocs}/learn/about-nextstrain.html`,



I deviated from this in some cases where it seems like it gets folks to a more relevant place in docs.nextstrain.org.

Reply:
Nice work!